example/multi-jvm-example/scripts/startPolicyService.sh

```diff
@@ -3,6 +3,6 @@
 java -jar example/multi-jvm-example/multi-jvm-example-rest-policy-service/target/multi-jvm-example-rest-policy-service-*-executable.jar \
                 -httpPort=8081 \
                 -extractDirectory=.extract/Policy \
-                -Dpalisade.rest.config.path=example/example-model/src/main/resources/configRest.json \
```

Comment:
For the localhost multi-jvm example I would rather it not be reliant on any of the docker services module, given that module will be dependent on multi-jvm example services modules